src/api/public/apidocs-new/paths/source_branch.yaml

```diff
@@ -0,0 +1,110 @@
+post:
+  summary: Branch a set of packages based on attributes
+  description: |
+    Create a source link from a package of an existing project to a
+    new subproject of the requesters home project.
+  security:
+    - basic_authentication: []
+  parameters:
+    - name: project
+      description: The project that you want to branch
+      in: query
+      schema:
+        type: string
```

Comment:
We have the [project name](https://github.com/openSUSE/open-build-service/blob/master/src/api/public/apidocs-new/components/parameters/project_name.yaml) for this, right?

Reply:
[project name](https://github.com/openSUSE/open-build-service/blob/master/src/api/public/apidocs-new/components/parameters/project_name.yaml) is different. The name of the required parameter in the API is `project` and it's a query parameter. The one you mentioned is defined as a path parameter.